### PR TITLE
front: fix stdcm GET departure time

### LINKED ISSUE #8310: GET: 2h offset with the trains in stdcm GET
### What happened?

I created a scenario with only one train, leaving at 12:09+:
image

I asked for a STDCM leaving between 08:00 and 10:00:
image

Which lead to a train leaving at 08:00 cause the grid's almost empty:
image

But i got a GET with `Train 1` leavin at 10:09 and `Train 2` leaving at 06:00:
image

It seems like there's a 2h difference

### What did you expect to happen?

.

### How can we reproduce it (as minimally and precisely as possible)?

.

### On which environments the bug occurs?

Recette (SNCF)

### On which browser the bug occurs?

Google Chrome

### OSRD version (top right corner `Account` button > `Informations`)

0a34296

### PR DESCRIPTION
**Pull request by Akctarus**
closes #8310

### CODE REVIEW

File: front/src/utils/date.ts [hunk 1]
```diff
@@ -177,6 +177,13 @@ export function convertIsoUtcToLocalTime(isoUtcString: string): string {
   return dayjs(isoUtcString).local().format();
 }
 
+export function convertToLocalTime(isoString: string): string {
```
Comment (emersion) [inline front/src/utils/date.ts]: I'd prefer not to add this function to `date.ts`: it forces UTC without converting. It's not something that one should do in general (although here it's necessary to workaround a space time chart bug).

Comment (bloussou) [reply front/src/utils/date.ts]: Yes and there should be no need to convert all the trains since only the stdcm is off

Comment (Akctarus) [reply front/src/utils/date.ts]: You're right, there was a much simpler way to fix this bug

**Review by bloussou — APPROVED** [review]
Tested !

**Review by emersion — APPROVED** [review]

**Review by kmer2016 — APPROVED** [review]
LGTM

**Pull request merged**